data/mods/Xedra_Evolved/eocs/item_eocs.json

```diff
@@ -23,5 +23,16 @@
     },
     "effect": [ { "u_message": "local_files_xedra", "snippet": true, "same_snippet": true, "popup": true } ],
     "false_effect": [ { "u_message": "You found nothing interesting." } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_WOLF_SLEDGE_TRHOW",
+    "condition": { "compare_num": [ { "u_val": "stamina" }, ">=", { "const": 3000 } ] },
+    "effect": [
+      { "u_assign_activity": "ACT_GENERIC_EOC", "duration": "2 seconds" },
```

Comment:
Throwing by default cost 0.6 seconds, and decreases up to 0.4 seconds at skill level 10, isn't 2 seconds is too much? but maybe it's not, since it's throwing and then returning an item 🤔